packages/semcom-demo-pods/data/metadata/.acl

```diff
@@ -0,0 +1,20 @@
+@prefix  acl:   <http://www.w3.org/ns/auth/acl#>.
+@prefix  foaf:  <http://xmlns.com/foaf/0.1/>.
+
+<#tony>
+    a               acl:Authorization;
+    acl:agent       <https://pods.actueel-inkomen.digita.ai/tiapieters/profile/card#me>;
+    acl:accessTo    <./>;
+    acl:default     <./>;
+    acl:mode        acl:Read,
+                    acl:Write,
+                        acl:Append,
+                        acl:Delete,
+                    acl:Control.
```

Comment:
We need to split this: authorization inside the folder should be read and write, but authorization on the folder itself should only be Append. None should include Control.